src/microsoft/office/license/microsoft-office-license.service.js

```diff
@@ -92,6 +92,16 @@ angular.module("Module.microsoft.services").service("MicrosoftOfficeLicenseServi
             .catch((err) => this.$q.reject(err.data));
     }
 
+    /**
+     * Get licence model
+     * @return {Object} [description]
```

Comment:
`[description]` => `Price with it's currency and textual representation` (according to API order.Price.description.

Reply:
This has been updated. Thanks for pointing this out :)